pandas/core/dtypes/concat.py

```diff
@@ -97,7 +97,9 @@ def _get_frame_result_type(result, objs):
     otherwise, return 1st obj
     """
 
-    if result.blocks and all(b.is_sparse for b in result.blocks):
+    if (result.blocks and (
+            all(is_sparse(b) for b in result.blocks) or
```

Comment:
related to my comment above. cannot is_sparse not simply check if its an EA and if it has a Sparse Dtype? 

then you simply need to pass the ``b.values`` here, yes?

Reply:
I'll give that a shot.

Reply:
I think a comment of mine may have been lost.

This is hit in several places (e.g. `pandas/tests/sparse/test_combine_concat.py::TestSparseDataFrameConcat::test_concat`).

What part can I clarify here?